Translation corrections (#74)
07th-mod/higurashi-console-arcs: Update/some_003.txt

			NULL, "\"This old man... he's a little strange.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamase1012", 256, TRUE);
	OutputLine(NULL, "どうも人付き合いが苦手な人みたいで、なかなかみんなと打ち解けてくれなくてね…』",

> 			NULL, " It seems he's not very good at communicating, and he doesn't really talk with everyone...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " It seems he's not very good at communicating, and he doesn't really chat with everyone...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>畠山老</color>", NULL, "<color=#f5e6d3>Hatakeyama</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_hatakeg1020", 256, TRUE);
	OutputLine(NULL, "「まぁ、都会じゃと何かと物入りで、息子の稼ぎだけでは苦しいからの。",

> 			NULL, "\"Well, in this city with all its expenses,  painful relying on only my son's earnings.", Line_WaitForInput);

Author: Sorry, messed up here. It should be "Well, in this city with all its expenses, it's hard for my son to live on his earnings alone."

> 			NULL, "\"Well, in this city with all its expenses,  painful relying on only my son's earnings.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Well, in this city with all its expenses, it's hard for my son to live on his earnings alone.", Line_WaitForInput);
```

//rだけど同時に、…村のお年寄りにとっては冗談や軽口、まして存在の否定を口にすることは絶対のタブーだと、これまでの体験から骨身にしみている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だけど同時に、…村のお年寄りにとっては冗談や軽口、まして存在の否定を口にすることは絶対のタブーだと、これまでの体験から骨身にしみている。",

> 			NULL, "However, I've learned from experience that the elders in the village consider any joking, slander, and disbelief of Oyashiro-sama to be absolutely taboo.", Line_Normal);

Reviewer: ```suggestion
			NULL, "However, I've learned from experience that the elders in the village consider any joking, slander, and disbelief of Oyashiro-sama to be an absolute taboo.", Line_Normal);
```

//rただ、…小さな不安だけは、ちくりと針で突かれたように、私の胸の中に残っていた…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ただ、…小さな不安だけは、ちくりと針で突かれたように、私の胸の中に残っていた…。",

> 			NULL, "But... a slight uneasiness remained in my chest,  piercing me like a needle...", Line_Normal);

Reviewer: ```suggestion
			NULL, "But... a slight uneasiness remained in my chest, piercing me like a needle...", Line_Normal);
```

//r甘い匂いに頬をゆるませながら、私は面映い思いでカップに口をつける。…甘くて、とってもおいしかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "甘い匂いに頬をゆるませながら、私は面映い思いでカップに口をつける。…甘くて、とってもおいしかった。",

> 			NULL, "The sweet smell made me smile as I I bashfully lifted the cup to my mouth. ...It was sweet, and very tasty.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The sweet smell made me smile as I bashfully lifted the cup to my mouth. ...It was sweet, and very tasty.", Line_Normal);
```

//r私はとっさに、『面会時間外だから使わせてもらった』と言い訳を考え、後ろめたい思いだったけど藤堂くんにそう説明しようと口を開きかけた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私はとっさに、『面会時間外だから使わせてもらった』と言い訳を考え、後ろめたい思いだったけど藤堂くんにそう説明しようと口を開きかけた。",

> 			NULL, "I thought of the excuse \"I used it because it's past of visiting hours.\" I felt guilty, but opened my mouth to explain it to Toudou-kun.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I thought of the excuse \"I used it because it's past visiting hours.\" I felt guilty, but opened my mouth to explain it to Toudou-kun.", Line_Normal);
```
